src/main/java/com/moodybluez/enterprise/dao/EntryDAO.java

```diff
@@ -0,0 +1,70 @@
+package com.moodybluez.enterprise.dao;
+
+import com.moodybluez.enterprise.dto.Entry;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+@Component
+public class EntryDAO implements IEntryDAO{
+
+    private Map<Integer, Entry> entries = new HashMap<>();
+
+    @Override
+    public boolean saveEntry(Entry entry) {
+        entries.put(entry.getEntryID(), entry);
+        if (entries.containsKey(entry.getEntryID())) return true;
```

Comment:
Ditto comment above.  You don't need the if test here.  containsKey() gives you the true/false result you want, so just return that.